Ginger/GingerCore/GingerCore.csproj

```diff
@@ -62,6 +62,9 @@
     <Reference Include="ALM_CommonStd">
       <HintPath>DLLs\ALM_CommonStd.dll</HintPath>
     </Reference>
+    <Reference Include="Ginger.ExecuterService.Contracts">
```

Comment:
Open PR for Executor Service Contracts project and use the nugget

Reply:
Removed